components/sup/src/census.rs

```diff
@@ -85,6 +90,7 @@ impl CensusRing {
            || (election_update_rumors.get_update_counter() > self.last_election_update_counter)
            || (service_config_rumors.get_update_counter() > self.last_service_config_counter)
            || (service_file_rumors.get_update_counter() > self.last_service_file_counter)
+           || (service_health_rumors.get_update_counter() > self.last_service_health_counter)
```

Comment:
Given the potential frequency and volume of health checks, I do not think it is feasible to use the same logic for service health rumors.

Reply:
What are your specific concerns? (Not doubting it, just asking for clarity)